packages/story-editor/src/components/panels/design/link/karma/link.karma.js

```diff
@@ -209,30 +209,39 @@ describe('Link Panel', () => {
       await setPageAttachmentLink('http://pageattachment.com');
     });
 
-    it('should not allow adding link in Page Attachment area', async () => {
+    fit('should not allow adding link in Page Attachment area', async () => {
       const insertElement = await fixture.renderHook(() => useInsertElement());
       const element = await fixture.act(() =>
         insertElement('shape', {
           backgroundColor: createSolidFromString('#ff00ff'),
           mask: { type: 'rectangle' },
-          x: 10,
-          y: 10,
+          x: 100,
+          y: 100,
           width: 50,
           height: 50,
         })
       );
       const frame = fixture.editor.canvas.framesLayer.frame(element.id).node;
-      await moveElementToBottom(frame);
+      await moveElementToBottom(frame, 100);
 
+      fixture.editor.inspector.designPanel.border.width().scrollTo(300, 300);
+      await fixture.events.sleep(1000);
+      await fixture.snapshot('Testing 1');
       linkPanel = fixture.editor.inspector.designPanel.link;
+      await linkPanel.address.scrollTo(50, 50);
```

Comment:
Hm, for some reason this scrolling here is just not working -- it is locally. Trying to scroll to the relevant input where the error message should show up.

@barklund @swissspidy Do you happen to have any idea why the `scrollTo` would not work when running in the CI? Perhaps I'm missing something obvious.

Reply:
Hm, actually, I could also try to bring it visible by closing the other panels, still odd that it doesn't work here though 🤔